ci-operator/step-registry/ipi/conf/openstack/generateconfig/ipi-conf-openstack-generateconfig-commands.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+#Set necessary variables
+CLUSTER_NAME=$(<"${SHARED_DIR}"/CLUSTER_NAME)
+OS_CLOUD=$(<"${SHARED_DIR}"/OS_CLOUD)
+OPENSTACK_EXTERNAL_NETWORK=$(<"${SHARED_DIR}"/OPENSTACK_EXTERNAL_NETWORK)
+OPENSTACK_COMPUTE_FLAVOR=$(<"${SHARED_DIR}"/OPENSTACK_COMPUTE_FLAVOR)
+LB_FIP_IP=$(<"${SHARED_DIR}"/LB_FIP_IP)
+BASE_DOMAIN=$(<"${SHARED_DIR}"/BASE_DOMAIN)
+CONFIG_TYPE=$(<"${SHARED_DIR}"/CONFIG_TYPE)
+
+PULL_SECRET=$(<"${CLUSTER_PROFILE_DIR}"/pull-secret)
+SSH_PUB_KEY=$(<"${CLUSTER_PROFILE_DIR}"/ssh-publickey)
+
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+if [[ "${CONFIG_TYPE}" == "minimal" ]]; then
+cat >> "${CONFIG}" << EOF
+apiVersion: v1beta4
+        baseDomain: ${BASE_DOMAIN}
+        controlPlane:
+          name: master
+        metadata:
```

Comment:
Cluster name set in `ipi-conf` step

Reply:
looks like comment here [1] might be outdated. Need to verify before using ipi-conf
https://github.com/openshift/release/blob/master/ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e.yaml#L310

Reply:
Oh, I see, perhaps its worth updating `ipi-conf` as well? I don't think CI cluster need >14 chars names anyway